studio/data/organizations/organization-billing-subscription-preview.ts

```diff
@@ -0,0 +1,91 @@
+import { useQuery, UseQueryOptions } from '@tanstack/react-query'
+import { post } from 'lib/common/fetch'
```

Comment:
use new method from `data/fetchers` - these will be provide validations against what endpoints are available + parameters etc

if this is a new endpoint - start API with `npm run start:nodemon`, then run `npm run codegen` in studio to update the data/api.d.ts file, with provides all the validations